apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.html

```diff
@@ -94,113 +94,157 @@ <h2 class="font-16 strong">{{ localeData?.choose_plans }}</h2>
-                                                                        : 0
-                                                                }}
-                                                                {{ commonLocaleData?.app_duration?.yearly | lowercase }}
-                                                            </p>
+                                                            <div
+                                                                class="d-flex justify-content-center align-items-center"
+                                                                [ngClass]="{
+                                                                    'popular-txt':
+                                                                        popularPlan?.uniqueName === plan?.uniqueName
+                                                                }"
+                                                            >
+                                                                <span class="font-20">{{ localeData?.popular }}</span>
+                                                            </div>
                                                         </ng-container>
                                                         <ng-container
-                                                            *ngIf="firstStepForm.get('duration')?.value === 'MONTHLY'"
+                                                            *ngIf="!(popularPlan?.uniqueName === plan?.uniqueName)"
                                                         >
-                                                            <p class="mb-2">
-                                                                <s>
-                                                                    <span class="strong"
-                                                                        >{{ plan?.currency?.symbol
-                                                                        }}{{ plan?.monthlyAmount }}</span
-                                                                    >
-                                                                </s>
+                                                            <div class="popular-txt-height"></div>
+                                                        </ng-container>
+                                                        <div class="plan-content d-flex flex-column">
+                                                            <h2
+                                                                class="font-20 strong plan-title d-flex align-items-center justify-content-center"
+                                                            >
+                                                                {{ plan?.name }}
+                                                            </h2>
+                                                            <p class="font-15 mt-3 feature-txt">
```

Comment:
<p class="font-15 mt-3 feature-txt" ***ngIf="plan?.description"**>
 
 use ngIf in p tag is better that child element span because p tag has only one child so in case of false condition p will render but with empty value

Reply:
This will use for blank p tag